apps/labs/labs-mokainosspringboot/labs-mokainosspringboot.yaml

```diff
@@ -7,9 +7,14 @@ metadata:
 spec:
   releaseName: labs-mokainosspringboot
   chart:
-    git: git@github.com:hmcts/labs-mokainosspringboot
-    ref: master
-    path: charts/labs-mokainosspringboot
+    spec:
+      chart: ./charts/labs-mokainosspringboot
+      version: '4.0.x'
```

Comment:
is this right?

Reply:
I think it should be:

```
      sourceRef:
        kind: GitRepository
        name: hmcts-charts
        namespace: flux-system
```

Reply:
I have create a new gitrepository source

Reply:
shouldn't you use the hmcts-charts one?